solc/CommandLineParser.cpp

```diff
@@ -139,6 +140,7 @@ static map<InputMode, string> const g_inputModeName = {
 	{InputMode::StandardJson, "standard JSON"},
 	{InputMode::Linker, "linker"},
 	{InputMode::LanguageServer, "language server (LSP)"},
+	{InputMode::CompilerWithEvmAssemblyJsonImport, "assembler (EVM ASM JSON import)"},
```

Comment:
You should make up your mind whether it's a variant of the assembler or compiler mode :)

Actually, I think it's neither. I initially wanted to say that it should be compiler, but it has different outputs, does not accept all of the same flags and conceptually does not work at Solidity source level like the compiler does. Unlike AST import, which basically runs the compiler, just based on info from the AST, not command line and sources, your new mode works on a completely different level. If we supported assembling EVM asm sources, this would be its variant that simply accepts input in JSON format.

```suggestion
	{InputMode::EVMAssemblerJSON, "EVM assembler (JSON format)"},
```